### ISSUE TITLE
[core][Method not decompiled]Nested try catch decompilation failed

### ISSUE THREAD

**Issue by xxjy** [skylot/jadx#1374]
Nested try catch decompilation failed.

jadx version: 1.3.2
jar file: [test-lib.zip](https://github.com/skylot/jadx/files/8033673/test-lib.zip)
jadx error:
```
/*  JADX ERROR: JadxRuntimeException in pass: BlockProcessor
    jadx.core.utils.exceptions.JadxRuntimeException: Unreachable block: B:3:0x000f
        at jadx.core.dex.visitors.blocks.BlockProcessor.checkForUnreachableBlocks(BlockProcessor.java:86)
        at jadx.core.dex.visitors.blocks.BlockProcessor.processBlocksTree(BlockProcessor.java:52)
        at jadx.core.dex.visitors.blocks.BlockProcessor.visit(BlockProcessor.java:44)
    */
public void test() {
    /*
        r3 = this;
        r0 = 1000(0x3e8, double:4.94E-321)
        java.lang.Thread.sleep(r0)     // Catch: Exception -> 0x0017, Exception -> 0x000f
        r0 = 1000(0x3e8, double:4.94E-321)
        java.lang.Thread.sleep(r0)     // Catch: Exception -> 0x0017, Exception -> 0x000f
        goto L_0x001c
    L_0x000f:
        r4 = move-exception
        r0 = r4
        r0.printStackTrace()     // Catch: Exception -> 0x0017
        goto L_0x001c
    L_0x0017:
        r4 = move-exception
        r0 = r4
        r0.printStackTrace()
    L_0x001c:
        return
    */
    throw new UnsupportedOperationException("Method not decompiled: defpackage.Test.test():void");
}
```
java source code:
```
public void test() {
    try {
        try {
            //two instructions and above
            Thread.sleep(1000);
            Thread.sleep(1000);
        } catch (Exception e) {
            e.printStackTrace();
        }
    } catch (Exception e) {
        e.printStackTrace();
    }
}
```

**Comment by skylot** [comment]
@xxjy thanks for report and a fix!
Funny that this issue appear only for java-input plugins (direct java bytecode parser) because `dx` change code to one without nested try blocks:
```java
    public void test() {
        try {
            Thread.sleep(1000L);
            Thread.sleep(1000L);
        } catch (Exception e) {
            try {
                e.printStackTrace();
            } catch (Exception e2) {
                e2.printStackTrace();
            }
        }
    }
```

**Comment by skylot** [comment]
Fixed in PR #1375 by @xxjy
